### PR TITLE
Feature guide module

### PR DESCRIPTION
**Pull request by Ekman1991**

### CODE REVIEW

**Comment by Ekman1991** [comment]
screen shot 2017-04-21 at 15 21 40
screen shot 2017-04-21 at 15 21 34
screen shot 2017-04-21 at 15 21 22
screen shot 2017-04-21 at 15 21 08

**Comment by bellevik** [comment]
👍 Looks code to me, codewise. But can't get it to run on my computer, some issues with versions of different components again.

**Comment by bex1** [comment]
Will go through this tomorrow or monday. Thank you guys for all your effort 👍

File: src/app/components/pages/guide-detail-page/guide-detail-page.component.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  </div>
+
+  <div>
+    <h2>{{ 'guide.header' | translate }}</h2>
+  </div>
+</div>
+
+<div class="ui centered grid guide-detail-wrapper">
+  <div class="row guide-detail-top-image-wrapper">
+    <img class="guide-detail-image" src="http://sipxcom.org/wp-content/uploads/2017/03/programming_blog-400x400.jpg" />
```
Comment (buren) [inline src/app/components/pages/guide-detail-page/guide-detail-page.component.html:13]: TODO

File: src/app/components/pages/guide-detail-page/guide-detail-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          'url' : 'https://google.se'
+        },
+        {
+          'header' : 'Link2',
+          'text' : 'Link 2 text Link 2 text Link 2 text Link 2 text Link 2 text Link 2 text Link 2 text Link 2 text',
+          'img' : 'http://sipxcom.org/wp-content/uploads/2017/03/programming_blog-400x400.jpg',
+          'url' : 'https://gp.se'
+        }
+      ]
+    };
```
Comment (buren) [inline src/app/components/pages/guide-detail-page/guide-detail-page.component.ts:28]: TODO

File: src/app/components/pages/guide/guide-page.component.ts [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+        'date' : '25 April 2017',
+        'img' : 'http://sipxcom.org/wp-content/uploads/2017/03/programming_blog-400x400.jpg'
+      },
+      {
+        'headerTitle' : 'Test3',
+        'text' : 'This is the text This is the text This is the text This is the text This is the text',
+        'date' : '25 April 2017',
+        'img' : 'http://sipxcom.org/wp-content/uploads/2017/03/programming_blog-400x400.jpg'
+      }
+    ];
```
Comment (buren) [inline src/app/components/pages/guide/guide-page.component.ts:31]: TODO

**Comment by buren** [comment]
Thanks 👍looks like a good starting point!

**Comment by bex1** [comment]
@buren This feature is handed to a third party right? I don't think this code will reach the app and its not that useful to build upon when designs change.

**Pull request closed without merging**